forch/forchestrator.py

```diff
@@ -114,20 +114,8 @@ def initialize(self):
         self._initialized = True
 
     def _attempt_authenticator_initialise(self):
-        radius_info = self._config.get('radius_info')
-        if not radius_info:
-            return
-        radius_ip = radius_info.get('server_ip')
-        radius_port = radius_info.get('server_port')
-        secret = radius_info.get('secret')
-        if not (radius_ip and radius_port and secret):
-            LOGGER.warning('Invalid radius_info in config. \
-                           Radius IP: %s; Radius port: %s Secret present: %s',
-                           radius_ip, radius_port, bool(secret))
-            raise ConfigError
-        self._authenticator = Authenticator(radius_ip, radius_port, secret, self.handle_auth_result)
-        LOGGER.info('Created Authenticator module with radius IP %s and port %s.',
-                    radius_ip, radius_port)
+        auth_config = self._config.get('orchestration', {}).get('auth_config')
+        self._authenticator = Authenticator(auth_config, self.handle_auth_result)
```

Comment:
this is no longer "attempt" because it does it all the time -- however, as per my other comment I'd leave the function name as is, but only call Authenticator() if auth_config is defined.